src/bin/substudy.rs

```diff
@@ -21,7 +21,7 @@ Subtitle processing tools for students of foreign languages
 
 Usage: substudy clean <subs>
        substudy combine <foreign-subs> <native-subs>
-       substudy export csv <video> <foreign-subs> [<native-subs>]
+       substudy export csv [--limit=<secs>] <video> <foreign-subs> [<native-subs>]
        substudy export review <video> <foreign-subs> [<native-subs>]
        substudy export tracks <video> <foreign-subs>
```

Comment:
I didn't want to add it to the other exports before I knew if you'd even accept this PR, but I agree with you, it's much better to be consistent and not surprise/confuse users. So I've added it to the other 2 `export` subcommands.